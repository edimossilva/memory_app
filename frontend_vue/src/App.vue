<template>
  <div id="app" class="has-background-white">
    <main-menu v-show="isLogged"></main-menu>
    <router-view />
  </div>
</template>
<script>
import MainMenu from "@/components/menus/MainMenu.vue";
import { mapState } from "vuex";

export default {
  name: "App",
  components: { MainMenu },
  computed: {
    ...mapState(["isLogged"]),
  },
};
</script>
<style>
html,
body {
  height: 100%;
}
#app {
  font-family: Avenir, Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
  height: 100%;
}

#nav {
  padding: 30px;
}

#nav a {
  font-weight: bold;
  color: #2c3e50;
}

#nav a.router-link-exact-active {
  color: #42b983;
}
</style>
