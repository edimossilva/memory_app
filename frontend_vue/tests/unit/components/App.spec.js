import { shallowMount } from "@vue/test-utils";
import App from "@/App.vue";
import Vue from "vue";
import VueRouter from "vue-router";

Vue.use(VueRouter);
describe("App.vue", () => {
  it("Matchs Snapshot", () => {
    const wrapper = shallowMount(App);
    expect(wrapper).toMatchSnapshot();
  });
});
