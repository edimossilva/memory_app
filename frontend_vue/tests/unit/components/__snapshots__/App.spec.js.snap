// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue Matchs Snapshot 1`] = `
<div id="app">
  <div id="nav">
    <router-link-stub to="/" tag="a" ariacurrentvalue="page" event="click">Home</router-link-stub> |
    <router-link-stub to="/about" tag="a" ariacurrentvalue="page" event="click">Review app</router-link-stub>
  </div>
  <router-view-stub name="default"></router-view-stub>
</div>
`;
